hazelcast/src/main/java/com/hazelcast/scheduledexecutor/impl/ScheduledExecutorMemberBin.java

```diff
@@ -16,14 +16,19 @@
 
 package com.hazelcast.scheduledexecutor.impl;
 
-import com.hazelcast.config.ScheduledExecutorConfig;
 import com.hazelcast.logging.ILogger;
 import com.hazelcast.spi.impl.NodeEngine;
 import com.hazelcast.internal.util.ConstructorFunction;
 
 public class ScheduledExecutorMemberBin
         extends AbstractScheduledExecutorContainerHolder {
 
+    /**
+     * Capacity controlled by PER_NODE config in the service layer
+     * (if set, {@link DistributedScheduledExecutorService#ensurePerNodeCapacity(String)}).
+     */
+    private static final int CAPACITY_DISABLED = 0;
```

Comment:
Maybe make this a constant on some shared class so `ScheduledExecutorPartition#ScheduledExecutorPartition` and `ScheduledExecutorContainer#checkNotAtCapacity` can use it as a constant.